'''
Created on 15 oct. 2019

@author: Val
'''

RESEARCH_LOG = "pyGenealogicalTools"
RESEARCH_INFO = "Research Log automatically generated by pyGenealogicalTools as tracking and storing of obtained results of automatic search"

MATCH_PROFILE_ERROR = "Profile provided does not contain the information of the GENI link"
MATCH_GENI = "This match was found automatically by PyGenealogical tools. MATCHED ON="
MATCH_CONFLICT_TASK = "CHECK GENI MATCH"
MATCH_CONFLICT_URL_EXISTING = "CHECK EXISTING MATCH"
MATCH_CONFLICT_URL_MESSAGE = "Profile has an existing link when trying to match the profile "
MATCH_CONFLICT_INFO = "This profile does not match with the following geni profile, when should == "
MATCH_REVIEW_TASK_BEGIN = "CHECK DOUBLE "
MATCH_REVIEW_TASK_END = " MATCH"
MATCH_REVIEW_DETAILS = "The following profiles are duplicated linked "
MATCH_ADDING_PROFILES = "Adding the profiles "
MATCH_EXISTING_PROFILES = "Linking existing profiles "
MATCH_POTENTIAL_DUPLICATE = "CHECK DUPLICATE "
MATCH_POTENTIAL_EXISTING = "CHECK EXISTING "
MATCH_POTENTIAL_INFO_EXISTING = "Trying to add the existing profile "
MATCH_PREVIOUS_MATCH = " as has a previous match but it has not been possible to find a match."

WEB_DETECTED = "Web automatically detected by pyGenealogicalTools"

TOP_CLIMB = "The maximum level of generations have been reached at the generation "

PROCESS_MATCH_NUMBER_OF_IMPACTS_BEGIN = "The number of available matches in "
PROCESS_MATCH_NUMBER_OF_IMPACTS_END = " is "
PROCESS_ADD_PROFILE_BEGIN = "Adding the profile "
PROCESS_LINK_PROFILE_BEGIN = "Linking the existing profile "
PROCESS_ADD_PROFILE_END = " as a "
PROCESS_NO_ACCESS = " profile is skipped as there is no access to the data "

PROCESS_TASK_NAME = "CHECK EVENTS"
PROCESS_TASK_DETAILS = "Review date for search of event "